tests/unit/api/test_model.py

```diff
@@ -422,7 +422,9 @@ def test_validate_model_bad_args(client, building_motif):
 def test_test_model_against_shapes(client, building_motif, shacl_engine):
```

Comment:
I think this method and test should be renamed.

```suggestion
def test_validate_model_against_shapes(client, building_motif, shacl_engine):
```

Reply:
This method should be renamed to match and its test should be moved to `/dataclasses`. 

https://github.com/NREL/BuildingMOTIF/blob/849e672963cdba10d2389dfdc67849b9859d2ead/buildingmotif/dataclasses/model.py#L232-L239